tests/containers/encrypted/app-config-verity/Makefile

```diff
@@ -1,9 +1,8 @@
-
-SGXLKL_ROOT=../../../..
+include ../../../common.mk
 
 CC_APP=/usr/bin/python3
 CC_APP_CONFIG_TEMPLATE=sgxlkl-encrypted-python-config-template.json
-CC_APP_CONFIG=sgxlkl-encrypted-python-config.json.tmp
+CC_APP_CONFIG=sgxlkl-encrypted-python-config.json
```

Comment:
This was on purpose as the file is generated, to avoid accidental committing to git.

Reply:
Reverted.